<ui version="4.0" >
 <class>advancedTabOptions</class>
 <widget class="QWidget" name="advancedTabOptions" >
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QVBoxLayout" >
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QCheckBox" name="m_pShowMMBInTabs" >
       <property name="whatsThis" >
        <string>This will open a new tab instead of a new window in various situations, such as choosing a link or a folder with the middle mouse button.</string>
       </property>
       <property name="text" >
        <string>Open &amp;links in new tab instead of in new window</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="m_pDynamicTabbarHide" >
       <property name="whatsThis" >
        <string>This will display the tab bar only if there are two or more tabs. Otherwise it will always be displayed.</string>
       </property>
       <property name="text" >
        <string>Hide the tab bar when only one tab is open</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="m_pNewTabsInBackground" >
       <property name="whatsThis" >
        <string>This will open a new tab in the background, instead of in the foreground.</string>
       </property>
       <property name="text" >
        <string>O&amp;pen new tabs in the background</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="m_pOpenAfterCurrentPage" >
       <property name="whatsThis" >
        <string>This will open a new tab opened from a page after the current tab, instead of after the last tab.</string>
       </property>
       <property name="text" >
        <string>Open &amp;new tab after current tab</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="m_pTabConfirm" >
       <property name="whatsThis" >
        <string>This will ask you whether you are sure you want to close a window when it has multiple tabs opened in it.</string>
       </property>
       <property name="text" >
        <string>Confirm &amp;when closing windows with multiple tabs</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="m_pPermanentCloseButton" >
       <property name="whatsThis" >
        <string>This will display close buttons inside each tab instead of websites' icons.</string>
       </property>
       <property name="text" >
        <string>&amp;Show close button on tabs</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="m_pPopupsWithinTabs" >
       <property name="whatsThis" >
        <string>Whether or not JavaScript popups if allowed shall open in a new tab or in a new window.</string>
       </property>
       <property name="text" >
        <string>Open pop&amp;ups in new tab instead of in new window</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="m_pTabCloseActivatePrevious" >
       <property name="whatsThis" >
        <string>When checking this the previous used or opened tab will be activated when you close the current active tab instead of the one right to the current tab.</string>
       </property>
       <property name="text" >
        <string>Activate previous used tab when closing the current tab</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="m_pKonquerorTabforExternalURL" >
       <property name="whatsThis" >
        <string>When you click a URL in another KDE program or call kfmclient to open a URL, the current desktop will be searched for a non-minimized Konqueror and, if found, the URL opened as a new tab within it. Otherwise a new Konqueror window will be opened with the required URL.</string>
       </property>
       <property name="text" >
        <string>Open as tab in existing Konqueror when URL is called externally</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="m_pMiddleClickClose" >
       <property name="whatsThis" >
        <string>When you click on a tab using the middle mouse button or mouse wheel, it will close that tab. (Konqueror needs to be restarted for changes to take effect.)</string>
       </property>
       <property name="text" >
        <string>Middle-click on a tab to close it</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="3" margin="6" />
 <layoutfunction spacing="KDialog::spacingHint" margin="KDialog::marginHint" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections/>
</ui>
